client/app/scripts/superdesk-desks/views/content-expiry.html

```diff
@@ -0,0 +1,11 @@
+<label translate ng-if="preview">{{ContentExpiry.Header}}
```

Comment:
it would be better like `<label ng-if="preview">{{ ContentExpiry.Header | translate }}`